windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -17,6 +17,26 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-3-1-1"]
+== Release notes for Red Hat Windows Machine Config Operator 3.1.1
+
+Issued: 2021-12-07
+
+The WMCO 3.1.1 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:4710[RHBA-2021:4710].
+
+[id="wmco-3-1-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `windows-exporter` metrics endpoint object contained a reference to a deleted machine. This incorrect reference caused the WMCO to ignore delete events for machines with invalid IP addresses. This bug fix removes the validation of the machine object from the event filtering, allowing the `windows-exporter` metrics endpoint object to correctly update when the machine is still in the `Deleting` phase. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008994[**BZ#2008994**])
```

Comment:
```suggestion
* Previously, the `windows-exporter` metrics endpoint object contained a reference to a deleted machine. This incorrect reference caused the WMCO to ignore deleted events for machines with invalid IP addresses. This bug fix removes the validation of the machine object from the event filtering, allowing the `windows-exporter` metrics endpoint object to correctly update when the machine is still in the `Deleting` phase. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008994[**BZ#2008994**])
```

Assuming this was intended to be "deleted" and not "delete". If my suggestion is wrong, please ignore.